WDL/CLI.py

```diff
@@ -2056,6 +2060,54 @@ def zip_wdl(
         Zip.build(doc, output, logger, meta=meta, inputs=input_dict, archive_format=fmt)
 
 
+def fill_generate_input_subparser(subparsers):
+    inputs_parser = subparsers.add_parser("inputs")
+    inputs_parser.add_argument(
+        "uri", metavar="WDL_URI", type=str, nargs="?", help="WDL document filename/URI"
+    )
+    return inputs_parser
+
+
+def generate_inputs(
+    uri=None,
+    path=None,
+    check_quant=True,
+    no_outside_imports=False,
+    **kwargs,
+) -> str:
+    """Print and returns a json string representing the required inputs for a wdl file.
+
+    Currently, only work for wdl file that contains a workflow and does not destruct wdl structs into json objects
```

Comment:
To support individual tasks, we could take a `--task` argument that can select one task in the WDL file if it has multiple tasks and no workflow. There is similar logic at the beginning of `runner_input()` we can copy or refactor into a helper function.

> and does not destruct wdl structs into json objects

I didn't quite follow this, and it seems like the struct template support is good!

Reply:
Sure will take a look at adding `--task` options and also see what i can reuse/refactor in runner_input() .

Can how i check about types like Pairs/Maps and Array?

ie 


```
struct Person {
  String Name
}
```

if there is an `Array[Person] group`,
should we return 
```
{"group" : [ {"name": "String"}]} 
{"group" : "Array(Person)"}